doc/examples/quickstart.py

```diff
@@ -1,3 +1,7 @@
+"""
+This tutorial is relevant for the On-Prem Exasol database.
```

Comment:
should we call it `example` rather than `tutorial`? I feel a tutorial usually is more extensive.